genie-agent/build.gradle

```diff
@@ -10,10 +10,13 @@ dependencies {
 
     // Spring
     compile("org.springframework.boot:spring-boot-starter")
+    compile("org.springframework.cloud:spring-cloud-aws-context")
+    compile("org.springframework.cloud:spring-cloud-starter-aws")
```

Comment:
@mprimi I think we only need to pull this in and it will pull your other AWS dependencies you've added here in transitively? Can check and fix in another commit.

Reply:
The STS one was not coming in transitively, but not even sure you need it. 
Not sure what's the minimum set you need, i added those to the patch just to save you a look up but we may need to revisit/change.